### PR TITLE
Replaced "is null" Type Checking with "== null" (#192)

### PR DESCRIPTION
**Pull request by TheClonker**
Hey, wanted to get my feet wet with s&box and C# after some experience with GMod (TTT) Addons.

This should cover all `is null` checks currently in the Code as described in Issue #192. For reference i just checked against the following RegExp `/is\s+(?:not\s+)?null/`.

I think it would be a good idea to enforce this in the CI, but i don't know enough about dotnet-format if it could be enforced with that alone. And I didn't wanna tinker with the config without your input 😃

### CODE REVIEW

**Review by Alf21 — APPROVED** [review]
Nice, thanks! Regarding CI / Linter / editconfigs, maybe @Histalek can provide further details / implement this.

**Comment by Histalek** [comment]
https://github.com/TTTReborn/tttreborn/blob/e41601c832dc6d4eb85975dfd46c387875e54ba5/.editorconfig#L66 should be the line that needs changing. To enforce this in CI it should be enough to set it to `false:warning`.

**Comment by TheClonker** [comment]
Hey, thanks for the quick response 🙂 @Histalek @Alf21

I updated the .editorconfig line but wasn't able to verify that `xyz is null` was marked as warning. 
But that could be an error on my side because i didn't setup Visual Studio correctly yet. Maybe one of you can verify that on your side ?

**Comment by mzegar** [comment]
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0041

I don't think there is an editor config for what we want.

File: .editorconfig [hunk 1]
```diff
@@ -63,7 +63,7 @@ dotnet_style_prefer_conditional_expression_over_assignment = true:silent
 dotnet_style_prefer_conditional_expression_over_return = true:silent
 dotnet_style_prefer_inferred_anonymous_type_member_names = true:suggestion
 dotnet_style_prefer_inferred_tuple_names = true:suggestion
-dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
+dotnet_style_prefer_is_null_check_over_reference_equality_method = false:warning
```
Comment (mzegar) [inline .editorconfig:66]: Change this to `true` and should be good to merge.

**Comment by Histalek** [comment]
> [docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0041](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0041)
> 
> I don't think there is an editor config for what we want. We would want to change the value to `true` anyways.

whoops you are right. It is the only config that mentions 'is null' so i simply assumed it will do what we want here

File: .editorconfig (hunk 1 above)
Comment (TheClonker) [reply .editorconfig]: Done

**Review by mzegar — APPROVED** [review]

**Pull request merged**